packages/prediction-app/src/components/BacktestsTable/BacktestsTable.tsx

```diff
@@ -0,0 +1,235 @@
+import React from 'react';
+import {
+    DataTable,
+    DataTableHead,
+    DataTableRow,
+    DataTableBody,
+    DataTableCell,
+    DataTableColumnHeader,
+    Checkbox,
+    Button,
+    IconAdd16,
+    DataTableFoot,
+    Pagination,
+    Chip,
+} from '@dhis2/ui';
+import i18n from '@dhis2/d2-i18n';
+import {
+    createColumnHelper,
+    flexRender,
+    getCoreRowModel,
+    useReactTable,
+    getSortedRowModel,
+    getFilteredRowModel,
+    getPaginationRowModel,
+} from '@tanstack/react-table';
+import { BackTestRead, ModelSpecRead } from '@dhis2-chap/chap-lib';
+import { useNavigate } from 'react-router-dom';
+import styles from './BacktestsTable.module.css';
+import { BacktestActionsMenu } from './BacktestActionsMenu';
+import { BacktestsTableFilters } from './BacktestsTableFilters';
+import { BatchActions } from './BatchActions';
+
+const statuses = {
+    COMPLETED: 'completed',
+    RUNNING: 'running',
+    FAILED: 'failed',
+}
+
+const labelByStatus = {
+    [statuses.COMPLETED]: i18n.t('Completed'),
+    [statuses.RUNNING]: i18n.t('Running'),
+    [statuses.FAILED]: i18n.t('Failed'),
+}
+
+const columnHelper = createColumnHelper<BackTestRead & { status?: string }>();
```

Comment:
Could we create a common type for the table object, and the reuse it in eg `BacktestsTableFilters`? 
```suggestion
// or some other name
export type BackTestWithStatus = BackTestRead & { status?: string }
export type BackTestReactTable = Table<BackTestWithStatus>

const columnHelper = createColumnHelper<BackTestWithStatus>();
```